packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -27,6 +27,38 @@ const defaultItemToString = item => {
 
 const getInstanceId = setupGetInstanceId();
 
+export const getA11yStatusMessageFunction = itemToCategory => {
+  // Copied directly from https://github.com/downshift-js/downshift/blob/v1.31.16/src/utils.js#L190
+  // to allow overwriting of aria-labels when categories are present
+  return ({
+    isOpen,
+    highlightedItem,
+    selectedItem,
+    resultCount,
+    previousResultCount,
+    itemToString,
+  }) => {
+    if (!isOpen) {
+      if (selectedItem) {
+        return itemToString(selectedItem);
+      } else {
+        return '';
+      }
+    }
+    const resultCountChanged = resultCount !== previousResultCount;
+    if (!resultCount) {
+      return 'No results.';
```

Comment:
Can you make this translatable, with [`translateWithId` interface](https://github.com/carbon-design-system/carbon/blob/v10.7.0/packages/react/src/components/Dropdown/Dropdown.js#L94-L97)? Ref: https://github.com/carbon-design-system/carbon/blob/v10.7.0/packages/react/docs/g11n.md

Reply:
see other comment